lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -223,19 +223,30 @@ def register_additional_file_watchers(global_state:, outgoing_queue:)
                 id: "workspace/didChangeWatchedFilesRails",
                 method: "workspace/didChangeWatchedFiles",
                 register_options: Interface::DidChangeWatchedFilesRegistrationOptions.new(
-                  watchers: [
-                    Interface::FileSystemWatcher.new(
-                      glob_pattern: "**/*structure.sql",
-                      kind: Constant::WatchKind::CREATE | Constant::WatchKind::CHANGE | Constant::WatchKind::DELETE,
-                    ),
-                  ],
+                  watchers: [structure_sql_file_watcher, yaml_file_watcher],
                 ),
               ),
             ],
           ),
         )
       end
 
+      sig { returns(Interface::FileSystemWatcher) }
+      def structure_sql_file_watcher
+        Interface::FileSystemWatcher.new(
+          glob_pattern: "**/*structure.sql",
+          kind: Constant::WatchKind::CREATE | Constant::WatchKind::CHANGE | Constant::WatchKind::DELETE,
+        )
+      end
+
+      sig { returns(Interface::FileSystemWatcher) }
+      def yaml_file_watcher
+        Interface::FileSystemWatcher.new(
+          glob_pattern: "**/*.{yml,yaml}",
```

Comment:
Let's limit the YAML files we watch to fixtures for now. We don't have any features planned that would depend on watching any other YAML files, so let's avoid increasing the load on the watcher and on the LSP.

Also, I think Rails allows configuring the fixture path, so you may need to start returning more information when booting the runtime server and using the configured paths to register these watchers.